packages/commonwealth/server/migrations/20241126161826-fix-LPBondingCurve-abi.js

```diff
@@ -0,0 +1,344 @@
+'use strict';
```

Comment:
Can you check if we need to re-order this migration

Reply:
I will just re-order to most todays date